spec/support/capybara.rb

```diff
@@ -9,15 +9,15 @@
   Capybara.default_max_wait_time = 4
   Capybara.javascript_driver = :chrome_en
 
-  port = ENV.fetch('CAPYBARA_SERVER_PORT', '0').to_i
+  port = ENV.fetch('CAPYBARA_SERVER_PORT', ENV.fetch('TEST_ENV_NUMBER', '1').to_i + 3000).to_i
   if port > 0
     Capybara.server_port = port
   end
   Capybara.always_include_port = true
 
   ip_address = Socket.ip_address_list.find { |ai| ai.ipv4? && !ai.ipv4_loopback? }.ip_address
   hostname = ENV['CAPYBARA_DYNAMIC_HOSTNAME'].present? ? ip_address : ENV.fetch('CAPYBARA_APP_HOSTNAME', 'localhost')
-  Capybara.server_host = '0.0.0.0'
+  Capybara.server_host = ip_address
```

Comment:
@oliverguenther might be the culprit as to why Travis fails to run the tests?

Reply:
Might be, you can use `ENV['TRAVIS']` to explicitly set 0.0.0.0 again in that case